### PR TITLE
workflow updates

### PR DESCRIPTION
**Pull request by sg-**
Docs and example program to use standard blinky which could be better done as any bin but that is for later

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,88 +1,60 @@
...
 Image: .\BUILD\<TARGET_NAME>\ARM\mbed-os-example-bootloader.bin
 ```
 
-It creates two binary files. The original uncombined image is in the output directory, <project-name>_application.bin, and the bootloader image is <project-name>.bin.
+This creates two binary files. The original unmodified image is in the output directory, <project-name>_application.bin, and the padded bootloader image is <project-name>.bin.
 
-When the build succeeds, you have created a bootloader for your target. This bootloader defines the update file to be located at `/sd/mbed-os-example-bootloader-blinky_application.bin`. This is the binary bootloader flashes and then jumps to.
+Copy the <project-name>.bin image to your board and without an SD card present or application file loaded on the SD card you should see an output similar to:
+```
+???What does this example print out???
```
Comment (0xc0170) [inline README.md:50]: should print by default without any app on the sd card:

```
    printf("No update found to apply\r\n");
    printf("Starting application\r\n");

```

**Comment by 0xc0170** [comment]
+1

File: README.md (hunk 1 above)
Comment (c1728p9) [reply README.md:50]: This bootloader example doesn't verify the application before starting it, so if the application is missing the device will crash and repeatedly print out "No update found to apply" and "Starting application".

If there are directions to load just the bootloader (and not the composite image) then the code should probably be updated to check for a missing application.

**Comment by bridadan** [comment]
@sg- @0xc0170 Almost all of these changes were made in #22. Do you want to close this?

**Pull request closed without merging**